lib/foreman_maintain/runner.rb

```diff
@@ -38,5 +43,23 @@ def ask_about_offered_steps(steps)
         @reporter.on_next_steps(self, steps)
       end
     end
+
+    def mock_skip_class(step)
+      Class.new(step.class.superclass) do
```

Comment:
Probably no use case for now, we can remove it add add it later when the use case is there